javascript-and-json/main.js

```diff
@@ -0,0 +1,32 @@
+var library = [
+  {
+    isbn: '123',
+    title: 'Amazing Ants',
+    author: 'Amy A'
+  },
+  {
+    isbn: '456',
+    title: 'Bewidering Beasts',
+    author: 'Billy B'
+  },
+  {
+    isbn: '789',
+    title: 'Challenging Cheetas',
+    author: 'Charly C'
+  }
+];
+
+console.log('Library:', library);
+console.log('Library Typeof:', typeof (library));
+
+var stingLibrary = JSON.stringify(library);
+console.log('stingLibrary:', stingLibrary);
+console.log('stingLibrary Typeof:', typeof (library));
+
+var student = JSON.stringify({ ID: '1185', name: 'Jimmy John' });
```

Comment:
You don't want to use `stringify` here, you want to write the object out as a string in the same format that a stringified JSON object is in